packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -20,17 +20,12 @@ class CreateCommand extends Command {
   final String name = 'create';
 
   @override
-  final String description = 'Create a new Flutter project.';
+  final String description = 'Create a new Flutter project.\nIf run on a project that already exists, this will repair the project, recreating any files that are missing.';
```

Comment:
You may want to move the additional description to a `final String usageFooter` section; it'll keep the help overview for the general flutter tool terse. We do this for trace and one or two other commands.